src/vshaxe/commands/Commands.hx

```diff
@@ -18,6 +18,7 @@ class Commands {
 		context.registerHaxeCommand(RunGlobalDiagnostics, server.runGlobalDiagnostics);
 		context.registerHaxeCommand(ToggleCodeLens, toggleCodeLens);
 		context.registerHaxeCommand(DebugSelectedConfiguration, debugSelectedConfiguration);
+		context.registerHaxeCommand(Type, extendedTyping);
```

Comment:
There's currently nothing that prevents this from always getting executed even for non-Haxe files once vshaxe has been activated.

I'm not sure if declaring it in `package.json` with a narrow `when` scope similar to that latex onEnter command I linked would do anything though, since it's a built-in command. If not we'd have to check the language ID of the active document in the callback. Also, I wonder if it would interfere with the Vim extension.